packages/ocaml-rocksdb/ocaml-rocksdb.0.02/opam

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
Normally we avoid having ocaml as part of the name of packages. See https://github.com/ocaml/opam-repository/tree/master/governance/policies#7-new-package-names-should-avoid-the-ocaml-prefixsuffix

I don't think this porject necessarily needs an exception. (Unless you can make a case for it?)

Reply:
I'm happy to make it "rocksdb" ?  Or any other name you would propose *grin*.